CHANGES.txt

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ----------
 
+Experimental Features
+~~~~~~~~~~~~~~~~~~~~~
+
+These features are experimental and may change at any point in the future.
+
+- The cookie API's now have the ability to set the SameSite attribute in both
```

Comment:
APIs (this is neither possessive nor a conjunction)

Also see following comments about "SameSite" and "samesite".